### PR TITLE
1.8 PlayerList fix

### PR DESCRIPTION
**Pull request by sanjay900**

### CODE REVIEW

**Comment by Bastikeks** [comment]
I think it's back.... 
citizens-2.0.14-Build1210
spigot 1.8 - build ?? from 19. february 2015

**Comment by Hergarian** [comment]
citizens-2.0.15 Preview 1.8.3 
Tablist is complete full of npc

File: src/main/java/net/citizensnpcs/npc/entity/EntityHumanNPC.java [hunk 1]
```diff
@@ -153,9 +158,7 @@ public void h() {
             // gravity. also works around an entity.onGround not updating issue
             // (onGround is normally updated by the client)
         }
-        if (!npc.data().get("removefromplayerlist", Setting.REMOVE_PLAYERS_FROM_PLAYER_LIST.asBoolean())) {
-            i();// TODO
-        }
```
Comment (YourSenpaiOnly) [inline src/main/java/net/citizensnpcs/npc/entity/EntityHumanNPC.java]: Is this what makes the players show up in tab?

**Comment by YourSenpaiOnly** [comment]
In 1.10.2 and the npc names still flicker into the tab list.

**Comment by sanjay900** [comment]
Honestly. this was an old fix. Go start an issue or something.

**Comment by YourSenpaiOnly** [comment]
So there is no fix?

**Comment by sanjay900** [comment]
I wouldn't know, haven't played with citizens for a long time.

**Pull request merged**